forui/lib/src/widgets/header/header_action.dart

```diff
@@ -0,0 +1,83 @@
+part of 'header.dart';
+
+/// Represents a [FHeader]'s action.
+class FHeaderAction extends StatelessWidget {
+  /// The icon;
+  final SvgAsset icon;
+
+  /// The callback.
+  ///
+  /// If null, the action is disabled.
+  final VoidCallback? onPress;
+
+  /// The style.
+  final FHeaderActionStyle? style;
+
+  /// Creates a [FHeaderAction].
+  const FHeaderAction({required this.icon, required this.onPress, this.style, super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final style = this.style ?? context.theme.headerStyle.actionStyle;
+
+    return Padding(
+      padding: style.padding,
+      child: FTappable(
+        onTap: onPress,
+        child: icon(
+          height: style.iconSize,
+          colorFilter: ColorFilter.mode(onPress == null ? style.disabledIcon : style.enabledIcon, BlendMode.srcIn),
+        ),
+      ),
+    );
+  }
+
+  @override
+  void debugFillProperties(DiagnosticPropertiesBuilder properties) {
+    super.debugFillProperties(properties);
+    properties
+      ..add(DiagnosticsProperty('icon', icon))
```

Comment:
Remind me to make `Asset`s mix-in `Diagnosticable`. Otherwise it'll throw an error.

Reply:
Should we create an issue for this?